src/validation/validation.cpp

```diff
@@ -2042,8 +2042,9 @@ bool ConnectBlockDependencyOrdering(const CBlock &block,
                     {
                         return false;
                     }
-                    return state.DoS(100, error("ConnectBlockDTOR(): inputs missing/spent"), REJECT_INVALID,
-                        "bad-txns-inputs-missingorspent");
+                    return state.DoS(100, error("ConnectBlockDTOR(): %s inputs missing/spent in tx %d %s",
```

Comment:
As you are at it, maybe add the word `block` here as well, so that it reads `block %s inputs missing/spent ...` ?

Minor nit though.

Reply:
makes sense will do